test.js

```diff
@@ -7,20 +8,25 @@ it('should return the name of named functions', function () {
 });
 
 it('anonymous', function () {
+	/* eslint-disable */
```

Comment:
Like https://github.com/kevva/download/blob/ab37da7408655bbdf87c2c4e186738c92a8707d7/test/test.js#L17, `eslint-disable-line` would make it simpler.

```js
assert.equal(fnName(function() {}), null); // eslint-disable-line
```